tests/rip7560/test_send_failed.py

```diff
@@ -126,6 +126,16 @@ def test_eth_send_account_validation_reverts1(
     assert_rpc_error(response, case.expected_message, -32000, expected_data)
 
 
+def encode_panic_error(w3, value):
```

Comment:
this is not a "panic" Panic(uint256), but "standard error" Error(string)

Reply:
Fixed.